src/main/java/com/minecolonies/coremod/commands/citizencommands/ListCitizensCommand.java

```diff
@@ -59,37 +67,72 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
         return super.getCommandUsage(sender) + "<colonyId>";
     }
 
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final List<ActionArgument> actionArgumentList,
+            @NotNull final Map<String, Object> argumentValueByActionArgumentNameMap) throws CommandException
+    {
+        Integer page = null;
+        final Object pageObject = argumentValueByActionArgumentNameMap.get("page");
+        if (null != pageObject)
+        {
+            page = (Integer) pageObject;
+        }
+        Colony colony = null;
+        final Object colonyObject = argumentValueByActionArgumentNameMap.get("colony");
+        if (null != colonyObject)
+        {
+            colony = (Colony) colonyObject;
+        }
+        executeShared(server, sender, colony, page);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "executeShared()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)